src/components/CommentList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Comment } from './Comment';
+
+export function CommentList({ comments }) {
+  return (
+    <div>
+      {comments.map(comment => (
+        <Comment comment={comment} key={comment.id} />
```

Comment:
Better to use spread operator here:
```suggestion
        <Comment {...comment} key={comment.id} />
```